templates/cluster/aws.json.tpl

```diff
@@ -0,0 +1,24 @@
+{
+    "manifest": {
```

Comment:
we're creating duplicate files, this can be avoided.

Reply:
these files look identical for now but it can change later if we introduce types that are very specific to cloud.